lib/commands/publish.js

```diff
@@ -127,6 +127,15 @@ class Publish extends BaseCommand {
     const noCreds = !(creds.token || creds.username || creds.certfile && creds.keyfile)
     const outputRegistry = replaceInfo(registry)
 
+    // if a workspace package is marked private then we skip it
+    if (workspace && manifest.private) {
```

Comment:
Is there a better way to check if the package is workspace package ?

Reply:
No this is fine.